CMakeLists.txt

```diff
@@ -176,9 +176,15 @@ if(PLATFORM_WIN32 OR PLATFORM_UNIVERSAL_WINDOWS)
     endif()
 endif()
 
-if(${CMAKE_GENERATOR} MATCHES "MinGW")
-    message("Building with MinGW")
-    set(MINGW_BUILD TRUE CACHE INTERNAL "Building with MinGW")
+set(MINGW_BUILD FALSE CACHE INTERNAL "MinGW build flag")
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
This is incorrect. You can cross-compile for Windows on Linux with MinGW. Try to google it.